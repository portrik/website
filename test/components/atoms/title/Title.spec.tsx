import { render } from '@testing-library/react';

import { Title } from '@components';

describe('Title', () => {
	test('Should Render Title', () => {
		const title = render(<Title>Test Title</Title>);

		expect(title.baseElement).toMatchSnapshot();
	});
});
