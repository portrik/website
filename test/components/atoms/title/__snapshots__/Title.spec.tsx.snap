// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Title > Should Render Title 1`] = `
<body>
  <div>
    <h1
      class="_title_17d4f4"
    >
      Test Title
    </h1>
  </div>
</body>
`;
